documentation/sphinx/source/mr-status-json-schemas.rst.inc

```diff
@@ -857,6 +857,20 @@
          "backup_worker_enabled":1,
          "perpetual_storage_wiggle": 0,
          "perpetual_storage_wiggle_locality":"0",
+         "storage_engine":{
+         "$enum":[
```

Comment:
Did you intent perpetual_storage_wiggle_engine or storage_engine?

Reply:
Good catch. Will fix it in a follow up PR since this one has passed CI...